Fixing NC and Invoice error when secuence is the same

SaulOrte opened the pull request:
[FIX #8597]
Problema: Se presenta error debido a una restriccion a nivel de base de datos en accoun.move donde el numero de secuencia no puede ser el mismo para notas de debido, credito o factura incluso al ser de diferentes compañias.
solucion: Se Elimino la restriccion SQL con un postinit y se creo nueva que q valida q el nombre o secuencia sea unico dependiendo del move_type en account_move, la compañia y el name

https://binaural.odoo.com/web#id=8597&cids=2&menu_id=302&action=386&model=helpdesk.ticket&view_type=form
 TICKET[X]

SaulOrte committed:
Fixing NC and Invoice error when secuence is the same

christopherBinaural reviewed (changes requested):
Deberias moverlo a un migration para que esas sentencias sql que eliminan cosas solo se ejecuten una vez y no cada vez que se actualiza el modulo

SaulOrte committed:
[FIX #8597]l10n_ve_accountant
En este Committ, se reestructura el codigo para hacerlo compatible como una migracion del codigo, esto evita
que la alteracion de la regla sql se ejecute ante cada actualizacion o instalacion del modulo.

https://binaural.odoo.com/web\#id\=8597\&cids\=2\&menu_id\=302\&action\=386\&model\=helpdesk.ticket\&view_type\=form
TICKET[X]

SaulOrte committed:
Merge branch '17.0' into 17.0_fix_ti-8597_fixing_error_when_creating_nc_and_invoice_with_same_secuence

SaulOrte committed:
Merge branch '17.0' into 17.0_fix_ti-8597_fixing_error_when_creating_nc_and_invoice_with_same_secuence

SaulOrte committed:
updating version migration
